app/controllers/V2App.scala

```diff
@@ -47,14 +51,17 @@ class V2App(isDev: Boolean, val acl: Acl, dynamoClient: DynamoDbClient, val deps
     val maybeUserData: Option[UserData] = Scanamo(dynamoClient).exec(
       userDataTable.get("email" === userEmail)).flatMap(_.toOption)
 
-    val clipboardArticles = if (editingEdition)
-      maybeUserData.map(_.editionsClipboardArticles.getOrElse(List()).map(ClipboardCard.apply))
-    else
+    val clipboardCards = if (editingEdition) {
+      if(isFeast)
+        maybeUserData.map(_.feastEditionsClipboardCards.getOrElse(List()).map(ClipboardCard.apply))
+      else
+        maybeUserData.map(_.editionsClipboardArticles.getOrElse(List()).map(ClipboardCard.apply))
+    } else
       maybeUserData.map(_.clipboardArticles.getOrElse(List()).map(ClipboardCard.apply))
```

Comment:
Non-blocking – when there's some logic spread over a few variables, one pattern is to use a tuple and pattern-match on the possibilities. This can look a bit more readable:

```suggestion
    val clipboardCards = (editingEdition, isFeast) match {
      case (true, true) =>  maybeUserData.map(_.feastEditionsClipboardCards.getOrElse(List()).map(ClipboardCard.apply))
      case (true, false) => maybeUserData.map(_.editionsClipboardArticles.getOrElse(List()).map(ClipboardCard.apply))
      case _ => maybeUserData.map(_.clipboardArticles.getOrElse(List()).map(ClipboardCard.apply))
    }
```

Anyhow, take a look and see what you think. (I've tested this locally, but do go ahead and make sure it is working yourself before merging!)